Othuum Responsiveness and Range Buff

lL1l1 opened the pull request:
## Description of the proposed changes
<!-- A clear and concise description (or visuals) of what the changes imply. -->
<!-- If it closes an issue, make sure to link the issue by using "(Closes/Fixes/Resolves) #(Issue Number)" in your pull request. -->
Goal: make it not get outranged as much and better to micro so that it is more viable in T3 raids/rushes.

- Increase Othuum's Bolters' range from 20 -> 22
- Increase Othuum turn rate 75 -> 130
- Increase Othuum max brake 2.85 -> 3.55. This slightly helps it turn from a standstill

https://github.com/FAForever/fa/assets/82986251/b7d2a285-d24b-459e-acf8-3789ca4afd6f


https://github.com/FAForever/fa/assets/82986251/e7ff4b66-1b5c-46cb-867b-1699dcfea531



## Additional context
Made by Balance Team request.

## Checklist

- [ ] Changes are documented in the changelog for the next game version

lL1l1 committed:
Update UnitBlueprint annotations

lL1l1 committed:
Minor fixes for othuum

Remove friendly fire from Thau Cannon
Fix asymmetrical firing arcs for Bolters
Remove unused RotateOnSpotThreshold

lL1l1 committed:
Increase Othuum MaxBrake

lL1l1 committed:
Give Othuum Harbinger turn rate

lL1l1 committed:
Increase Othuum's Bolters' range 20 -> 22

Garanas reviewed (changes requested):
It is a big sturdy unit on tracks - with this turn rate it feels weightless. I'd rather we explore different directions before making the Othuum feel weightless.

lL1l1 committed:
Reduce turn speed buff

lL1l1 committed:
Increase turret yaw speed as needed

1.2x of turnrate for thau cannon, 1.6x of turnrate for bolters

lL1l1 committed:
Remove BackUpDistance

it causes backwards movement on nearby move orders unintentionally, and isn't needed as much with the turn rate buffed

lL1l1 committed:
Add changelog

lL1l1 commented:
https://github.com/FAForever/fa/assets/82986251/be456668-55ad-4a99-adce-e3cb64a8cdda

MrRowey commented:
Please can you change the `TurnRate` to `90`

MrRowey committed:
Merge branch 'deploy/fafdevelop' into balance/othuum-turning
